src/OrchardCore.Modules/OrchardCore.Contents/Controllers/ApiController.cs

```diff
@@ -121,7 +122,7 @@ public async Task<IActionResult> Post(ContentItem model, bool draft = false)
                     return this.ChallengeOrForbid();
                 }
 
-                contentItem.Merge(model);
+                contentItem.Merge(model, new JsonMergeSettings { MergeArrayHandling = MergeArrayHandling.Replace });
```

Comment:
Store it in static filed instead of new up on every call

```csharp
private static readonly JsonMergeSettings _jsonMergeSettings = new JsonMergeSettings { MergeArrayHandling = MergeArrayHandling.Replace };
```